solidity/test/ConverterUpgrader.js

```diff
@@ -40,7 +40,7 @@ describe('ConverterUpgrader', () => {
         const upgrader = await ConverterUpgrader.new(contractRegistry.address);
 
         await contractRegistry.registerAddress(registry.CONVERTER_UPGRADER, upgrader.address);
-        if (version) {
+        if (version && version < 45) {
```

Comment:
why < 45 here? if any, probably <= 45, but I think that we can still use addConnector anyway as we still have backward compatibility for it

Reply:
1. We no longer have `addConnector` in neither `ConverterBase` or `StandardPoolConveter` :(
2. Since we don't have `addConnector` in version 45, I had to exclude it explicitly. The actual version it was removed from is <45, but since we only check 23 and 45 - excluding 45 works as well.

Reply:
sounds good